src/app/containers/Ad/MPU.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_TRPL,
+  GEL_SPACING_QUAD,
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
+} from '@bbc/gel-foundations/spacings';
+import styled from 'styled-components';
+import BaseAdContainer from '.';
+
+const AdContainer = styled(BaseAdContainer)`
+  @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+    /* To remove GEL Margins */
```

Comment:
It is out of scope for this PR, but seems like the `AdContainer` should be modified to configurably remove the padding based on a prop rather than resorting to negative margins. This couples the MPU to the ad container's styling implmentation meaning any change to the `AdContainer` could break the styling of the MPU.

Reply:
I have added https://github.com/bbc/simorgh/issues/8025 to address this in future